java/bulk-import/bulk-import-starter/src/test/java/sleeper/bulkimport/starter/executor/EmrExecutorTest.java

```diff
@@ -81,13 +87,13 @@ public void shouldPrettyPrintTheJobToAvoidEmrBug() {
         // Then
         List<String> args = requested.get().getSteps().get(0).getHadoopJarStep().getArgs();
         String jobArg = args.get(args.size() - 2);
-        assertEquals("{\n"
+        assertThat(jobArg).isEqualTo("{\n"
                 + "  \"files\": [\n"
                 + "    \"file1.parquet\"\n"
                 + "  ],\n"
                 + "  \"id\": \"my-job\",\n"
                 + "  \"tableName\": \"myTable\"\n"
-                + "}", jobArg);
+                + "}");
```

Comment:
This test might be flaky due to JSON comparison? I noticed this was removed in the last merged PR.

Reply:
Looks like PR #102 changed this behaviour. I'm not sure if there should be a test to replace it.